src/tests/unittests/algorithms/test_fitting.py

```diff
@@ -4,10 +4,25 @@
 import numpy as np
 import matplotlib.pyplot as plt
 import qtt
-from qtt.algorithms.functions import FermiLinear, linear_function, double_gaussian, gaussian
+from qtt.algorithms.functions import FermiLinear, linear_function, double_gaussian, gaussian, sine
 from qtt.algorithms.fitting import initFermiLinear, _estimate_fermi_model_center_amplitude, fitFermiLinear,\
-        fit_double_gaussian, refit_double_gaussian, fit_gaussian
-
+        fit_double_gaussian, refit_double_gaussian, fit_gaussian, fit_sine
```

Comment:
A nice alternative is to put the imports together in parentheses instead of the end of line slash thingy.